apiary-metastore-listener/README.md

```diff
@@ -105,8 +107,9 @@ The following shows an example JSON message representing an "ALTER_PARTITION" ev
       "eventType": "ALTER_PARTITION",
       "dbName": "some_db",
       "tableName": "some_table",
-      "partition": ["col_1","col_2"],
-      "oldPartition": ["col_1", "col_2", "col_3"]
+      "partitionKeys": ["col_1","col_2"],
+      "partitionValues": ["val_1", "val_2", "val_3"],
```

Comment:
The number of values need to match the number of keys. If we're sending one event per alter (which is the case currently) then we just have a list like here.

Reply:
done.